packages/core/README.md

```diff
@@ -1,3 +1,157 @@
-# Slice Machine Core
+# Slicemachine Core
```

Comment:
I think this was up to date with Hugo's PR but we should update it.
A simple paragraph per folder you have in the Core is enough I think.
No need for available methods as it will evolve.